models/order/src/line-item-return-item/transformers.ts

```diff
@@ -0,0 +1,22 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TLineItemReturnItem, TLineItemReturnItemGraphql } from './types';
+
+const transformers = {
```

Comment:
It seems we're missing the `custom` property in the `buildFields` list of the transformers.